docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
When a section consists only of guides, start the name with a bare infinitive to indicate that. A good name for this section would be `Add Unzer credentials in the Back Office`.